pkg/config/template.go

```diff
@@ -242,9 +299,10 @@ log_level = "{{ .LogLevel }}"
 # This option supports live configuration reload.
 log_filter = "{{ .LogFilter }}"
 
-# Allow usage of the annotation io.kubernetes.cri-o.userns-mode
-# for setting up a user namespace.  This feature is experimental, do not use
-# in production.  It might be changed in future without notice.
+# Allow usage of the annotation io.kubernetes.cri-o.userns-mode for setting up a user namespace.
+# A runtime class must be configured with "runtime_allow_userns_annotation" for this option to
+# have any effect.
+# This feature is experimental, do not use in production. It might be changed in future without notice.
 allow_userns_annotation = {{ .AllowUsernsAnnotation }}
```

Comment:
Yeah I agree having nearly the same option will be misleading to the users. Can we remove/deprecate it?

Reply:
yes, if we support a finer control, let's just drop `allow_userns_annotation`.  Since it is experimental we can avoid the deprecation